{"artifact": "ensino/primeiro-semestre-de-2023/index.html", "exception": "AssertionError", "traceback": "Traceback (most recent call last):\n  File \"/home/gilberto/.local/pipx/venvs/lektor/lib/python3.10/site-packages/lektor/builder.py\", line 900, in update\n    yield ctx\n  File \"/home/gilberto/.local/pipx/venvs/lektor/lib/python3.10/site-packages/lektor/builder.py\", line 1151, in build_artifact\n    build_func(artifact)\n  File \"/home/gilberto/.local/pipx/venvs/lektor/lib/python3.10/site-packages/lektor/build_programs.py\", line 191, in build_artifact\n    artifact.render_template_into(self.source[\"_template\"], this=self.source)\n  File \"/home/gilberto/.local/pipx/venvs/lektor/lib/python3.10/site-packages/lektor/builder.py\", line 724, in render_template_into\n    rv = self.build_state.env.render_template(\n  File \"/home/gilberto/.local/pipx/venvs/lektor/lib/python3.10/site-packages/lektor/environment/__init__.py\", line 229, in render_template\n    return self.jinja_env.get_or_select_template(name).render(ctx)\n  File \"/home/gilberto/.local/pipx/venvs/lektor/lib/python3.10/site-packages/jinja2/environment.py\", line 1301, in render\n    self.environment.handle_exception()\n  File \"/home/gilberto/.local/pipx/venvs/lektor/lib/python3.10/site-packages/jinja2/environment.py\", line 936, in handle_exception\n    raise rewrite_traceback_stack(source=source)\n  File \"/home/gilberto/Dropbox/UFBA/pagina_pessoal/lektor/homepage/templates/acoesensino.html\", line 1, in top-level template code\n    {% extends \"layout.html\" %}\n  File \"/home/gilberto/Dropbox/UFBA/pagina_pessoal/lektor/homepage/templates/layout.html\", line 53, in top-level template code\n    {% block body %}{% endblock %}\n  File \"/home/gilberto/Dropbox/UFBA/pagina_pessoal/lektor/homepage/templates/acoesensino.html\", line 3, in block 'body'\n    <p> Publicado por <em>{{ this.author }}</em> em <em>{{ this.pub_date|dateformat('full') }} </em>.</p>\n  File \"/home/gilberto/.local/pipx/venvs/lektor/lib/python3.10/site-packages/lektor/environment/__init__.py\", line 36, in new_func\n    return func(*args, **kwargs)\n  File \"/home/gilberto/.local/pipx/venvs/lektor/lib/python3.10/site-packages/babel/dates.py\", line 730, in format_date\n    return pattern.apply(date, locale)\n  File \"/home/gilberto/.local/pipx/venvs/lektor/lib/python3.10/site-packages/babel/dates.py\", line 1308, in apply\n    return self % DateTimeFormat(datetime, locale)\n  File \"/home/gilberto/.local/pipx/venvs/lektor/lib/python3.10/site-packages/babel/dates.py\", line 1314, in __init__\n    assert isinstance(value, (date, datetime, time))\nAssertionError"}
